gir_ffi.gemspec

```
@@ -22,12 +22,7 @@ Gem::Specification.new do |s|
     provide enough or correct information to create sane bindings, overrides may be created.
   DESC
 
-  s.files = Dir["{lib,test,tasks,examples}/**/*",
-                "*.md",
-                "Gemfile",
-                "Rakefile",
-                "COPYING.LIB"] & `git ls-files -z`.split("\0")
-
+  s.files = Dir["{docs,examples,lib}/**/*", "COPYING.LIB"]
```

Hi @mvz,

> I agree that using git inside the gemspec is not very nice. However, one purpose of this is making sure no unwanted files (e.g., vim swap files) are shipped in the gem. Any thoughts on how alternative ways to handle this?

Yep, here's what you might be interested in:

```suggestion
  s.files = Rake::FileList["{docs,examples,lib}/**/*", "COPYING.LIB"].exclude(*File.read('.gitignore').split)
```

Let me know what you think!? :smile:

---

Ooh, nice!